docs/docs/testing.md

```diff
@@ -0,0 +1,92 @@
+---
+id: testing
+title: "Testing with Jest"
+sidebar_label: "Testing with Jest"
+---
+
+Reanimated test mocks use web implementation of Reanimated2. Before you begin using Reanimated mocks you need some setup actions.
+
+## Setup
+
+To your `jest-setup.js` file add the following line:
+```js
+require('./node_modules/react-native-reanimated/src/reanimated2/jestUtils').setUpTests();
+```
+`setUpTests({ fps: 60 })` can take optional config argument with specific frameRate. By default it is 60fps.  
+
+To be sure, check your `jest.config.js` file to contains:
+```js
+...
+preset: 'react-native',
+setupFiles: ['./jest-setup.js'],
+...
+```
+
+## API
+
+#### Style checker
+- Checking equality of selected styles with current component styles
+  #### `expect(component).toHaveAnimatedStyle(expectedStyle)`
+  `component: obj` - tested component  
+  `expectedStyle: obj ` - conntains expected styles of testing component, for example `{ width: 100 }`  
+
+- Checking equality all current component styles with expected styles
+  #### `expect(component).toHaveAnimatedStyle(expectedStyle, true)`
```

Comment:
What do you think about this API?

I prefer explicit parameter names and this would allow us to extend the API in the future if needed.

```suggestion
  #### `expect(component).toHaveAnimatedStyle(expectedStyle, {exact: true})`
```

Reply:
Good idea 👍